### PR TITLE
Update python/pip/setuptools just enough to fix webpy builds

### PR DESCRIPTION
**Pull request by danielfm**
This image was already upload with the tags `0.4.1` and `0.4.1-onbuild`.

### CODE REVIEW

File: onbuild/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jusbrasil/python-web:0.5
+FROM jusbrasil/python-web:0.4.1
```
Comment (felipemsantana) [inline onbuild/Dockerfile:1]: Why are you downgrading version?

Comment (danielfm) [reply onbuild/Dockerfile:1]: The 0.5 version was my previous attempt, which apparently broke webpy. The current image in production is 0.4, so I'm creating 0.4.1 now

Comment (felipemsantana) [reply onbuild/Dockerfile:1]: In the future someone could bump this to 0.5 and the push/build will break.
It's better to bump to 0.5.1, unless you delete 0.5 from Docker registry.

Comment (danielfm) [reply onbuild/Dockerfile:1]: I prefer to delete the 0.5 from the registry, to make sure no one tries to use it until we sort this out.

Comment (danielfm) [reply onbuild/Dockerfile:1]: Done, these tags are no longer in our registry

**Review by felipemsantana — APPROVED** [review]

**Comment by danielfm** [comment]
I'll keep this open until we get the change applied to webpy and we are certain it doesn't break anything. If this happens (let's hope so!), I'll merge this PR and tag the new version.

**Comment by danielfm** [comment]
Webpy broke again. Will delete these tags from the Docker registry.

**Pull request merged**